internal/commands/config_prune_interval.go

```diff
@@ -0,0 +1,105 @@
+package commands
+
+import (
+	"encoding/json"
+	"fmt"
+	"regexp"
+
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+
+	"github.com/buildpacks/pack/internal/config"
+	"github.com/buildpacks/pack/internal/style"
+	"github.com/buildpacks/pack/pkg/image"
+	"github.com/buildpacks/pack/pkg/logging"
+)
+
+func ConfigPruneInterval(logger logging.Logger, cfg config.Config, cfgPath string) *cobra.Command {
```

Comment:
I think you can use 
```go
cfg config.Config 
cfgPath string
```
To get the path where you want to save the `image.json`,  `cfgPath` has the path to the `config.toml` file, then you can use `filepath.Dir(cfgPath)` to get the parent directory.